api/cpp/CMakeLists.txt

```diff
@@ -158,6 +162,19 @@ if (SLINT_BUILD_RUNTIME)
         target_compile_options(Slint INTERFACE /bigobj)
     endif()
 
+    foreach(feature ${public_cmake_features})
+        set(cmake_option "SLINT_FEATURE_${feature}")
+        if(${cmake_option})
+            list(APPEND enabled_features ${feature})
+        else()
+            list(APPEND disabled_features ${feature})
+        endif()
+    endforeach()
+    set_property(TARGET Slint APPEND PROPERTY SLINT_ENABLED_FEATURES "${enabled_features}")
+    set_property(TARGET Slint APPEND PROPERTY SLINT_DISABLED_FEATURES "${disabled_features}")
+    set_property(TARGET Slint APPEND PROPERTY EXPORT_PROPERTIES "SLINT_ENABLED_FEATURES")
+    set_property(TARGET Slint APPEND PROPERTY EXPORT_PROPERTIES "SLINT_DISABLED_FEATURES")
```

Comment:
Do we need the SLINT_PREFIX on the property?

Should these target property be documented?

Reply:
I think a `SLINT` prefix is good practice to avoid any kind of conflicts. The target property namespace is shared with cmake, although it's unlikely that they'll start using `SLINT_` :-).

Yes, I can add that to the docs.